articles/sql-database/sql-database-automated-backups.md

```diff
@@ -63,13 +63,50 @@ If you delete a database, SQL Database keeps the backups in the same way it woul
 - **LTR**: SQL Database offers the option configuring long-term retention of full backups for up to 10 years. If LTR policy is enabled, theses backups are stored in RA-GRS storage automatically, but you can control how often the backups are copied. To meet different compliance requirement, you can select different retention periods for weekly, monthly and/or yearly backups. This configuration will define how much storage will be used for the LTR backups. You can use the LTR pricing calculator to estimate the cost of LTR storage. For more information, see [Long-term retention](sql-database-long-term-retention.md).
 
+## How to update backup retention period
+
+You can update backup retention period for a database between 7 and 35 days (during vCore-based purchasing model preview, the backup retention period is 7 days, and cannot be updated). You can update the backup retention period using REST API or PowerShell:
+
+### Update backup retention period to 14 days using REST API
+**Sample Request**
+```http
+PUT https://management.azure.com/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/resourceGroup/providers/Microsoft.Sql/servers/testserver/databases/testDatabase/backupShortTermRetentionPolicies/default?api-version=2017-03-01-preview
+```
+**Request Body**
+```json
+{
+  "retentionDays": 14
+}
+```
+**Sample Response**
+
+Status code: 200
+```json
+{
+  "id": "/subscriptions/00000000-1111-2222-3333-444444444444/providers/Microsoft.Sql/resourceGroups/resourceGroup/servers/testserver/databases/testDatabase/backupShortTermRetentionPolicies/default",
+  "name": "default",
+  "type": "Microsoft.Sql/resourceGroups/servers/databases/backupShortTermRetentionPolicies",
+  "properties": {
+    "retentionDays": 14
+  }
+}
+```
+See [Backup Short Term Retention Policies REST API documentation](austin-to-send-me-the-link) for more details.
+
+### Update backup retention period to 14 days using PowerShell (will be supported in Azure Powershell Vxxx.x.x)
+```powershell
+Set-AzureRmSqlDatabaseBackupShortTermRetentionPolicy -ResourceGroupName resourceGroup -ServerName testserver -DatabaseName testDatabase -RetentionDays 14
+```
+
+Updating backup retention period in Azure portal is not supported yet. 
+
 ## How to extend the backup retention period?
```

Comment:
Can we change this to be more clear between LTR and STR?

Reading "How to extend the backup retention period" I assumed it was about to discuss updating the short term retention

Reply:
added "automated backup" to distinguish from LTR